src/views/error-view-switcher/index.tsx

```diff
@@ -10,6 +10,8 @@ import { DevDot404, DevDotFallback, DevDotVersioned404 } from '../error-views'
 
 const VERSION_PATTERN = /\/(?<version>v\d+([.]|_)\d+([.]|_)(\d+|x))/ // matches semver with . and _ separators
```

Comment:
Looks like both of these are no longer used right?

Reply:
yes, correct! I removed them in the most recent commit